components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabConfig.kt

```diff
@@ -30,7 +29,6 @@ import java.util.UUID
  * @property externalAppType How this custom tab is being displayed.
  */
 data class CustomTabConfig(
-    val id: String = UUID.randomUUID().toString(),
```

Comment:
🤔 I do not remember why this was added. Probably when we initially migrated / introduced the store.

Reply:
Yeah, same, it seemed unnecessary nowadays and just made things more complicated (e.g. when comparing)